### PR TITLE
legacy_tunables: Pick up some modern defaults.

### PR DESCRIPTION
**Pull request by akshatsikarwar**
/silent

### CODE REVIEW

File: db/config.c [hunk 1]
```diff
@@ -394,9 +394,7 @@ static char *legacy_options[] = {
     "dont_superset_foreign_keys",
     "enable_sql_stmt_caching none",
     "enable_tagged_api",
-    "env_messages",
     "init_with_time_based_genids",
-    "legacy_schema on",
```
Comment (riverszhang89) [inline db/config.c:399]: Should we keep it on in 7.0, but repurpose the tunable for 7.1 features (e.g., uuid, autoincrement, etc.) in master/7.1 branch?

Comment (nirbhayc) [reply db/config.c:399]: Good idea. Another option would be to have individual tunables to control new features.

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: <b>Error</b>. &#x26A0;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Error &#x26A0;.<br>Regression testing: <b>1/347 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>recover_ddlk_sp</p>

File: db/config.c (hunk 1 above)
Comment (akshatsikarwar) [reply db/config.c:399]: I prefer new tunables for new features. Old tunables should control exactly what old branch did. Do not want to reuse these parameters.

Comment (riverszhang89) [reply db/config.c:399]: I thought you preferred no new tunables.

**Review by riverszhang89 — APPROVED** [review]

**Comment by akshatsikarwar** [comment]
Gonna wait a bit for the last of R6 to disappear.

**Comment by akshatsikarwar** [comment]
Thank you for the review @riverszhang89

**Comment by akshatsikarwar** [comment]
Only doing queuedb genid filenames: https://github.com/bloomberg/comdb2/pull/3295

**Pull request closed without merging**